tools/webpack-plugin/src/models/BacktracePluginOptions.ts

```diff
@@ -1,5 +1,7 @@
-import { DebugIdGenerator } from '@backtrace/sourcemap-tools';
+import { DebugIdGenerator, SourceMapUploader } from '@backtrace/sourcemap-tools';
 
 export interface BacktracePluginOptions {
     debugIdGenerator?: DebugIdGenerator;
+    sourceMapUploader?: SourceMapUploader;
```

Comment:
This is the customer-facing interface - can we describe all options and what it means if something is not set up - for example, "uploadUrl"? As a user, I'm not sure how the library will behave when uploadUrl is undefined.

Reply:
I added some docs.